api/kroxylicious-api/kroxylicious-api.iml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The phantom iml. We should revisit if we can switch to a more typical idea gitignore now.

Reply:
removed .iml from the PR.